tests/test_generic/tests/test_xml.py

```diff
@@ -431,6 +432,15 @@ def _check_is_published_false(self, xml_content, file_name):
         if '<record ' in xml_content and '<field name="is_published" eval="True"/>' in xml_content:
             _logger.warning(f"Model in {file_name} should not have 'is_published' set to True.")
 
+    def _check_website_published_false(self, xml_content, file_name):
+        if (
+            '<record ' in xml_content
+            and '<field name="website_published" eval="True"/>' in xml_content
+        ):
+            _logger.warning(
+                f"Model in {file_name} should not have 'website_published' set to True."
```

Comment:
Maybe you could be guiding a bit more, and provide a message mentioning that it is better to use `is_published` and only in demo (same comment for the test about is_published)